lib/interjectable.rb

```diff
@@ -34,8 +44,8 @@ def inject(dependency, &default_block)
     # Defines helper methods on instances that memoize values per-class.
     # (shared across all instances of a class, including instances of subclasses).
     #
-    # Calling a second time clears the value (if set) from previous times. This should
-    # probably only be called a second time in tests.
+    # Calling a second time is an error. Use `#test_inject` for overriding in
```

Comment:
Maybe mention "require the test helpers for RSpec and use test_inject" because otherwise it won't be defined, right?